docs/docs/sample-datasets.md

```diff
@@ -48,6 +48,26 @@ ds = xr.open_dataset(
 )
 ```
 
+<!-- === "Cloudflare R2" -->
+
+<!-- ```python -->
+<!-- import icechunk as ic -->
+<!-- import xarray as xr -->
+
+<!-- storage = ic.s3_storage( -->
+<!--     bucket="icechunk-public-data", -->
+<!--     prefix="v1/era5_weatherbench2", -->
+<!--     endpoint_url="foo", -->
```

Comment:
need to expose this publicly through a custom subdomain: https://developers.cloudflare.com/r2/buckets/public-buckets/#connect-a-bucket-to-a-custom-domain